wwwroot/config.json

```diff
@@ -156,7 +156,6 @@
       "trainerHeight": "64",
       "workbenchWidth": "350"
     },
-    "useCesiumIonBingImagery": true,
```

Comment:
When this line is removed, the error "Failed to add members from JSON for model basemaps" will occur when testing nm locally by running `yarn start` .
Maybe we should set "useCesiumIonBingImagery" to false in "privateclientconfig-2021-10-14.json" in aws?

Reply:
This is actually due to basemap config error and will be fixed in `natmap-catalog-v8`.